tests/test_plotting_cartoon.py

```diff
@@ -0,0 +1,126 @@
+"""Test plotting module."""
+
+import pytest
+
+import arviz as az
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+import xarray as xr
+
+import hssm
+from hssm.plotting.utils import (
+    _get_plotting_df,
+    _xarray_to_df,
+    _get_title,
+    _subset_df,
+    _row_mask_with_error,
+    _process_df_for_qp_plot,
+)
+from hssm.plotting.posterior_predictive import (
+    _plot_posterior_predictive_1D,
+    _plot_posterior_predictive_2D,
+    plot_posterior_predictive,
+)
+from hssm.plotting.quantile_probability import (
+    _plot_quantile_probability_1D,
+    _plot_quantile_probability_2D,
+    plot_quantile_probability,
+)
+
+hssm.set_floatX("float32")
+
+
+# I want to parameter
+@pytest.mark.parametrize(
+    ["n_trajectories", "groups", "plot_pp_mean", "plot_pp_samples", "row", "col"],
+    [
+        (2, None, False, False, "participant_id", "stim"),
+        (2, None, True, True, "participant_id", "stim"),
+        (2, None, False, True, "participant_id", "stim"),
+        (0, None, False, True, "participant_id", "stim"),
+        (0, None, True, False, "participant_id", "stim"),
+        (2, ["dbs"], True, True, "participant_id", "stim"),
+        (2, None, True, False, "participant_id", None),
+        (2, None, True, False, "participant_id", "stim"),
+        (2, None, True, False, None, None),
+    ],
+)
+def test_plot_model_cartoon_2_choice(
+    cav_model_cartoon, n_trajectories, groups, plot_pp_mean, plot_pp_samples, row, col
+):
+    """Test plot_model_cartoon for 2-choice data."""
+    if (not plot_pp_mean) and (not plot_pp_samples):
+        with pytest.raises(AssertionError):
```

Comment:
Again, I think `ValueError` conveys what the type of the error is and might be a better fit here